app/views/users/new.html.erb

```diff
@@ -0,0 +1,52 @@
+
+<%= form_for @user do |f| %>
+
+  <% @user.errors.full_messages.each do |error| %>
+  <h3><%= error %></h3>
+  <% end %>
+<p>
+  <%= f.label :username, :value => "your user here:" %> <br/>
+  <%= f.text_field :username, :placeholder => "your name" %>
+</p>
+<p>
+  <%= f.label :email, :value => "your email here:" %> <br/>
+  <%= f.text_field :email, :placeholder => "example@email.com" %>
+</p>
+<p>
+  <%= f.label :password, :value => "your password here:" %> <br/>
+  <%= f.password_field :password %>
+</p>
+<p><%= f.submit %></p>
+<% end %>
+
+<!--
+<%= form_tag("/users", method: "post") do %>
```

Comment:
Please, use **`users_path`** instead of `/users`

Reply:
ok i will take care thanks